lib/pqos.h

```diff
@@ -1026,6 +1031,7 @@ struct pqos_alloc_config {
         enum pqos_cdp_config l3_cdp;     /**< requested L3 CAT CDP config */
         enum pqos_cdp_config l2_cdp;     /**< requested L2 CAT CDP config */
         enum pqos_mba_config mba;        /**< requested MBA config */
+        enum pqos_mba_config smba;       /**< requested SMBA config */
```

Comment:
Add smba at the end

Reply:
Please use one of reserved fields for binary compability